### DISCUSSION TITLE
GoogleMaps API

### DISCUSSION THREAD

**Discussion by mariuszmalek** [brefphp/bref#951] [Support] [answered]
Can I connect to the GoogleMaps API? When I try through Laravel I get an error. I mean, I don't see an error in the logs, but the software is standing there and it crashes me.

**Comment by t-richard** [comment]
Is this from a Lambda or from the frontend ? 

If it's in Lambda then maybe you don't have internet access because you are inside a VPC and your Lambda does not have acces to internet.

If that's the case then generally your Lambda will timeout.

**Comment by mnapoli** [comment, marked as answer]
@mariuszmalek to be honest, it seems like you may struggling with many different issues, with Bref but maybe also more generally with Laravel and hosting in general (judging by the last 5 tickets you opened).

Maybe you might be better using something a bit simpler to host Laravel: have you looked at Laravel Forge for example?

I'm saying this because I sometimes see a few people struggle like this, and it's very likely that you may be wasting a lot of time when you could get things done faster with another approach. So I'd rather save you some time and frustration.

Hope that helps!

**Reply by t-richard** [reply]
I think Matthieu is right.

Bref is awesome, no doubts about that. But apart from that, it's still AWS' logic you have to figure out : VPCs, Subnets, Security Groups, RDS, Nat Gateways, SQS, S3, Cloudfront and how all those services are hooked up together to become an awesome infrastructure.

Either you truly want to figure all of that and go through the hassle of understanding all of that.

Or just want to get your site hosted somewhere with the less efforts, then go for Laravel Forge, maybe laravel Vapor or even general pupose PaaS like Heroku or Platform.sh

And if you want to stick with bref and AWS, then start simple. Just a simple laravel app printing Hello world, then add assets and make sure it works as expected, go through trial and error, read about AWS and so on.

Then go through the same process with adding a queue, putting your lambda in VPC, adding a database, adding internet access when inside a VPC, etc. Building something incremental and learning along the way.

Either way, I wish you luck :muscle:
